assets/js/src/common/grid/centered_row.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import classnames from 'classnames';
+
+type Props = {
+  children?: React.ReactNode;
+  className?: string;
+};
+
+const CenteredRow = ({ children, className }: Props) => (
+  <div className={classnames(className, 'mailpoet-centered-row')}>
+    {children}
+  </div>
+);
+
+export default CenteredRow;
```

Comment:
We want to avoid using export default, and we should use named exports everywhere.